mtp_cashbook/apps/mtp_auth/forms.py

```diff
@@ -0,0 +1,53 @@
+from django import forms
+from django.contrib.auth import authenticate
+
+from django.utils.translation import ugettext_lazy as _
+
+from .exceptions import ConnectionError
+
+
+class AuthenticationForm(forms.Form):
+    """
+    Authentication form used for authenticating users during the login process.
+    """
+    username = forms.CharField(label=_("Username"), max_length=254)
+    password = forms.CharField(label=_("Password"), widget=forms.PasswordInput)
+
+    error_messages = {
+        'invalid_login': _("Please enter a correct username and password. "
+                           "Note that both fields may be case-sensitive."),
+        'connection_error': _("The API Server seems down, please try again later."),
```

Comment:
Those are just keys as any other keys, you will be able to change them in the translation files if you want although I doubt we need translation files.

This is standard django convention, explicit english translation keys and error_messages hash.
I suggest we shouldn't change it as it would be too confusing for devs.